explorer/src/utils/nftoken-parsers.ts/isNFTokenAccount.ts

```diff
@@ -51,11 +58,14 @@ export const parseNFTokenCollectionAccount = (
   if (!parsed) {
     return null;
   }
+  if (Buffer.from(parsed.discriminator).toString("base64") !== "RQLwA3YS2fI=") {
+    return null;
+  }
 
   return {
     address: account!.pubkey.toBase58(),
     authority: parsed.authority,
     authority_can_update: Boolean(parsed.authority_can_update),
-    metadata_url: parsed.metadata_url,
+    metadata_url: parsed.metadata_url?.replace(/\0/g, "") ?? null,
```

Comment:
What's this?

Reply:
It strips null characters from the string due to the way that the Buffer string is parsed. I don't have to do this with `nftoken-js` since I have a smarter parser there.